tests/sample/map_reuse_2.c

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+// clang -O2 -Werror -c map_reuse_bad.c -o map_reuse_bad.o
+//
+// For bpf code: clang -target bpf -O2 -Werror -c map_reuse_bad.c -o map_reuse_bad.o
```

Comment:
make comments match filename

```suggestion
// clang -O2 -Werror -c map_reuse_2.c -o map_reuse_2.o
//
// For bpf code: clang -target bpf -O2 -Werror -c map_reuse_2.c -o map_reuse_2.o
```